pkgs/servers/routinator/default.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv, fetchFromGitHub, rustPlatform }:
+
+rustPlatform.buildRustPackage rec {
+  name = "routinator-${version}";
```

Comment:
Please replace with `pname = "routinator";` as `name = "${pname}-${version}";` is already given.

Reply:
[nixpkgs/manual](https://nixos.org/nixpkgs/manual/#compiling-rust-applications-with-cargo) should include this as well.
Thanks for your feedback.

Reply:
I have never packaged a rust application yet... I should really get around to doing so! Thanks :smile: